msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java

```diff
@@ -212,21 +212,21 @@ private void checkCurrentAccountNotifyCallback(@NonNull final CurrentAccountCall
         callback.onAccountLoaded(newAccount);
     }
 
-
     @Override
     public void signIn(@NonNull final Activity activity,
+                       @NonNull final String loginHint,
                        @NonNull final String[] scopes,
                        @NonNull final AuthenticationCallback callback) {
         acquireToken(
                 activity,
-                new String[]{"user.read"},
```

Comment:
was this an accidental change before to default this scope ?

Reply:
Probably accidental. Just caught it too.